src/Componenets/Header/Header.js

```diff
@@ -7,8 +7,43 @@ class Header extends Component {
 		return (
 			// Header goes here
 			<header>
-				{/* Write your HTML here */}
-				test header
+
+				{/* Adding container-fluid div to contain navbar*/}
+
+				<div className="container-fluid">
+					<nav className="navbar navbar-expand-md navbar-light bg-light">
+
+						{/* Remove text and add Logo image path later */}
+						<a className="navbar-brand" href="#"><img alt="" src="" />SKILL UP</a>
+
+						{ /* Adding Bootstrap 4 collapsible menu button for mobile devices */ }
+						<button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
This is not opening the menu to down when the screen size is small. Can you check it? You can have a look [here](https://codesandbox.io/s/github/abhishekakade/quiz-app).

Reply:
I know. I removed all the extra bootstrap classes bro since styling was gonna be another issue. Check the issue. @ezioda004 said no styling so I just made the skeleton. There's just minimum amount of CSS which I thought should remain so I left it. Actually removing the Bootstrap code/classNames took more time than other stuff :man_facepalming:

Reply:
If you want me to add that again, I can get it done soon. Let me know...

Reply:
Yeah, I know it surely had taken a lot of time. btw thanx for your contribution.
It's ok then. :innocent: